README.md

```diff
@@ -80,68 +36,35 @@ To build and deploy to GitHub Pages:
    - Paste the PAT from previous step into the Value box
      ![](static/img/secrets2.png)
 
-#### A Shortcut To Update Your gh-pages:
-
-To deploy your updates to Github Pages, run:
-
-- For Bash:
-
-```console
-   GIT_USER=<GITHUB_USERNAME> yarn deploy
-```
-
-- For Windows
-
-```console
-GIT_USER=<Your GitHub username> USE_SSH=true yarn deploy
-```
-
-- For Powershell
-
-```console
-cmd /C 'set "GIT_USER=<GITHUB_USERNAME>" && yarn deploy'
-```
-
-If you are using GitHub pages for hosting, this command is a convenient way to build the website and push to the `gh-pages` branch.
-
-## Miscellanous
-
-### Tutorial for Converting Table-like Diagrams into Images
-
-To prevent table diagrams overflowing (and having the horizontal scrollbar on narrower viewports),
-we use a simple technique to convert them to images. This allows us to have responsive diagrams
-with less effort and minimizes HTML syntax on our pages.
-
-Tool Recommendation: Firefox Browser (easier to make screenshots)
-
-1. Run the app (`npm run start`)
-
-2. Go to the browser
-
-3. Go where the table you want to convert is
-
-4. Right click, select "Take Screenshot"
-
-![](static/img/take-screenshot.png)
-
-5. Select an area that encloses the whole table
-
-![](static/img/click-to-select-region.png)
+3. Go to the Actions Tab
+   - You will likely see a red x beside a run labelled Initial commit
+   - Click the Initial Commit link
+   - Hit the rerun jobs button
 
-6. Make any adjustments necessary when cropping (avoid having unnecessary space)
+## Files that Require changes:
```

Comment:
Change title: 
## Files You Must Update:

Reply:
fixed